templates/InstallChocolatey.ps1.erb

```diff
@@ -63,13 +64,20 @@ if ($unzipMethod -eq '7zip') {
 }
 
 # call chocolatey install
-Write-Host "Installing chocolatey on this machine"
+Write-Output "Installing chocolatey on this machine"
 $toolsFolder = Join-Path $tempDir "tools"
 $chocInstallPS1 = Join-Path $toolsFolder "chocolateyInstall.ps1"
 
-& $chocInstallPS1
+if ($PSVersionTable.psversion.Major -gt 2) {
+  & $chocInstallPS1
+}
+else {
+  $output = Invoke-Expression $chocInstallPS1
+  $output
```

Comment:
you will see the output, but not the error log... if you want to see the errors, like on all the silentlycontinue stuff, it is going to throw a lot of people off, on perfectly fine installs due to inconsequentialities.  i tried on a dozen servers of varying configs, and think that if people have an issue with an install gone bad, they can be referred to the error log. 

We can add a write-output that states the error logs for PS2 version folks are found in $env:temp if they have any issues.